crates/uv-resolver/src/resolver/urls.rs

```diff
@@ -77,62 +67,6 @@ impl Urls {
             }
         }
 
-        // Add any editable requirements. If there are any conflicts, return an error.
-        for (editable, metadata) in &manifest.editables {
-            if let Some(previous) = required.insert(metadata.name.clone(), editable.url.clone()) {
-                if !is_equal(&previous, &editable.url) {
-                    if is_precise(&previous, &editable.url) {
-                        debug!(
-                            "Assuming {} is a precise variant of {previous}",
-                            editable.url
-                        );
-                        allowed.insert(editable.url.clone(), previous);
-                    } else {
-                        return Err(ResolveError::ConflictingUrlsDirect(
-                            metadata.name.clone(),
-                            previous.verbatim().to_string(),
-                            editable.verbatim().to_string(),
-                        ));
-                    }
-                }
-            }
-
-            for requirement in metadata
-                .requires_dist
-                .iter()
-                .filter(|requirement| requirement.evaluate_markers(markers, &editable.extras))
-            {
-                if let Some(pep508_rs::VersionOrUrl::Url(url)) = &requirement.version_or_url {
-                    if let Some(previous) = required.insert(requirement.name.clone(), url.clone()) {
-                        if !is_equal(&previous, url) {
-                            if is_precise(&previous, url) {
-                                debug!("Assuming {url} is a precise variant of {previous}");
-                                allowed.insert(url.clone(), previous);
-                            } else {
-                                return Err(ResolveError::ConflictingUrlsDirect(
-                                    requirement.name.clone(),
-                                    previous.verbatim().to_string(),
-                                    url.verbatim().to_string(),
-                                ));
-                            }
-                        }
-                    }
-                }
-            }
-        }
-
-        // Add any overrides. Conflicts here are fine, as the overrides are meant to be
-        // authoritative.
-        for requirement in manifest
-            .overrides
-            .requirements()
-            .filter(|requirement| requirement.evaluate_markers(markers, &[]))
-        {
-            if let Some(pep508_rs::VersionOrUrl::Url(url)) = &requirement.version_or_url {
-                required.insert(requirement.name.clone(), url.clone());
-            }
-        }
```

Comment:
Able to delete a ton of code here because overrides are now applied in the _iterator_, rather than needing a special case down here.